Makefile

```diff
@@ -191,7 +191,13 @@ license-verify:
 fmt:
 	@echo ">> Ensuring style of files"
 	@$(GOPATH)/bin/goimports -w $(SOURCES)
-	@$(GOPATH)/bin/gci write -s "standard" -s "default" -s "prefix(github.com/arangodb)" -s "prefix(github.com/arangodb/kube-arangodb)" $(SOURCES)  
+	@$(GOPATH)/bin/gci write -s "standard" -s "default" -s "prefix(github.com/arangodb)" -s "prefix(github.com/arangodb/kube-arangodb)" $(SOURCES) 
+
+.PHONY: fmt-generated
+fmt-generated:
```

Comment:
Why not format these files in same 'fmt' target?

Also, why not gormat all files under '$(ROOT)/pkg/generated/' ?
Adding specific directories makes makefile aware of too many things